cmd/talosctl/cmd/talos/list.go

```diff
@@ -148,13 +149,23 @@ var lsCmd = &cobra.Command{
 					}
 				}
 
-				fmt.Fprintf(w, "%s\t%s\t%d\t%d\t%s\t%s\t%s\n",
+				label := ""
+				if info.Xattrs != nil && len(info.Xattrs) > 0 {
+					for _, l := range info.Xattrs {
+						if l.Name == "security.selinux" {
+							label = string(l.Data)
```

Comment:
after this, maybe better to use `break`?
```go
        if l.Name == "security.selinux" {
            label = string(l.Data)
            break
        }
```
exit the loop once the desired Name is found, preventing unnecessary iterations.